static/src/javascripts/projects/common/modules/experiments/tests/facebook-share-params.js

```diff
@@ -0,0 +1,52 @@
+define([
+    'common/utils/config'
+], function (
+    config
+) {
+    return function () {
+        this.id = 'FacebookShareParams';
+        this.start = '2016-06-06';
+        this.expiry = '2016-06-13';
+        this.author = 'Kate Whalen';
+        this.description = 'Change the URL sent to the facebook crawler to manipulate FB caching';
+        this.audience = 0;
+        this.audienceOffset = 0;
+        this.successMeasure = 'We want to see how image branding affects Facebook shares and click-through';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = 'Open Graph';
+        this.idealOutcome = 'Increase in guardian traffic from facebook cards with image overlays';
+
+        this.canRun = function () {
+            return config.page.contentType == 'Article';
+        };
+
+        function alterCanonicalUrlSent(hrefContent, additionalParam) {
+            var urlhalves = hrefContent.split('&redirect_uri=');
```

Comment:
Just following this through but you're splitting at `&redirect_uri=` so you can get the `href=` value right? Do you need to urlEncode the `?` passed in here?